hw/ip/tlul/adapter_reg_racl.core

```diff
@@ -11,7 +11,8 @@ filesets:
       - lowrisc:prim:assert
       - lowrisc:prim:secded
       - lowrisc:tlul:common
-      - lowrisc:tlul:trans_intg
+      - lowrisc:tlul:adapter_reg
+      - lowrisc:tlul:request_loopback
```

Comment:
I think this looks reasonable to me. Am I right in thinking that we weren't actually using the `trans_intg` core directly even before this PR?

Reply:
Exactly. The ordinary reg adapter was using that and that one has a direct dependency.